README.md

```diff
@@ -86,9 +86,14 @@ For sorting imports and clean code with black run next command:
 To start auth-server run command:
 
 ```shell
-python3 -m sc_server_auth [-h] [-H HOST] [-p PORT] [-d DATABASE] [-l LOG_LEVEL] [-r] [-e DOT_ENV] [-g GOOGLE_SECRET_FILE_PATH]
+python -m sc_server_auth [-h] [-H HOST] [-p PORT] [-d DATABASE] [-l LOG_LEVEL] [-r] [-e DOT_ENV] [-g GOOGLE_SECRET_FILE_PATH]
 ```
 
+Google secret file you can get in [Google Developer Console](https://console.developers.google.com/)
+
+Creating project [tutorial](https://developers.google.com/workspace/guides/create-project)
+Giting client_secret.json file [tutorial](https://help.talend.com/r/en-US/7.2/google-drive/how-to-access-google-drive-using-client-secret-json-file-the)
```

Comment:
Ok, I got a secret file from Google via Credentials -> OAuth 2.0 Client IDs (after create an application - I am not sure that it is a correct way). I typed it as a param while running a server, and opened page 127.0.0.1:5000/docs. After that I get a token through auth/get_google_token. How do I check this new token?